scripts/core/registry.yml

```diff
@@ -211,6 +211,9 @@ etors:
 - name: MEM_GET_NATIVE_HANDLE
   desc: Enumerator for $xMemGetNativeHandle
   value: '68'
+- name: ENQUEUE_READ_HOST_PIPE
+  desc: Enumerator for $xEnqueueReadHostPipe
+  value: '69'
```

Comment:
[NIT suggestion] maybe it is better to move that at the end with `ENQUEUE_WRITE_HOST_PIPE` for more clarity, but not sure of that as that will leave the gap here empty.

Reply:
So this file gets populated automatically when you run the `generate` target, pretty strange why it's decided to put it there. I just tried removing both and it has added them in the same places. I don't think it's too important as long as they're both present in the ifle.

Reply:
Ah okay, I didn't notice that it is auto-generated, just ignore that.